src/pages/Report/Situation/index.view.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styled from 'styled-components';
+import LinkPreview from '../../components/LinkPreView';
+
+const Wrapper = styled.div`
+  position: relative;
+  display: flex;
+  margin: auto;
+  width: 100%;
+  height: 100%;
+  align-items: center;
+  min-width: 360px;
+  max-width: 90vw;
+`;
+
+const ListContent = styled.div`
+  width: 100%;
+  height: 100%;
+  color: black;
+  display: flex;
+  justify-content: center;
+  @media only screen and (max-width: 768px) {
+    margin-bottom: 10px;
+  }
+`;
+
+const NewsListWrapper = styled.div`
+  width: 100%;
+  @media only screen and (max-width: 1100px) {
+    margin-top: 65px;
+  }
+`;
+
+const ViewSituationNews = ({ news, onClickCard }) => {
+  return (
+    <Wrapper>
+      <NewsListWrapper>
+        {news.map((item, index) => {
+          return (
+            <ListContent key={index}>
```

Comment:
แนะนำให้ใช้ id หรืออะไรที่ unique จริวในการใส่ key นะครับ (ถ้ามี) ถ้าใช้ index อาจจะมี bug ได้ครับ

https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318